addon/adapters/drf.js

```diff
@@ -15,23 +15,8 @@ import Ember from 'ember';
  */
 export default DS.RESTAdapter.extend({
   defaultSerializer: "DS/djangoREST",
-
-  init: function() {
-    this._super();
-
-    if (this.get('coalesceFindRequests')) {
-      var error = "Please ensure coalesceFindRequests is not present or set " +
-        "to false in your adapter. This adapter does not support the " +
-        "coalesceFindRequests option. The Django REST Framework does not " +
-        "offer easy to configure support for N+1 query requests in the " +
-        "format that Ember Data uses (e.g. GET /comments?ids[]=1&ids[]=2) " +
-        "See the Ember documentation about coalesceFindRequests for " +
-        "information about this option: " +
-        "http://emberjs.com/api/data/classes/DS.RESTAdapter.html#property_coalesceFindRequests";
-      throw new Ember.Error(error);
-    }
-  },
```

Comment:
I think it would be a good idea to display a warning reminding users to implement the back-end properly when `coalesceFindRequests` is set to `true` (as you suggested in an email). There should also be a flag to disable the warning. Something like `CoalesceFilterBackendEnabled = true` could work but feel free to pick something you like (if you agree).